apiserver/src/main/java/org/dependencytrack/model/ConfigPropertyConstants.java

```diff
@@ -94,6 +94,7 @@ public enum ConfigPropertyConstants {
     GITLAB_SBOM_PUSH_ENABLED("integrations", "sbom.push.enabled", "true", PropertyType.BOOLEAN, "Flag to enable/disable SBOM generation for GitLab projects", ConfigPropertyAccessMode.READ_WRITE),
     GITLAB_TOPICS("integrations", "gitlab.topics", "[]", PropertyType.STRING, "JSON array of topics to include when syncing GitLab projects", ConfigPropertyAccessMode.READ_WRITE),
     GITLAB_URL("integrations", "gitlab.url", "https://gitlab.com", PropertyType.URL, "Base URL to GitLab instance", ConfigPropertyAccessMode.READ_WRITE),
+    GITLAB_JWKS_PATH("integrations", "gitlab.jwks.path", null, PropertyType.ENCRYPTEDSTRING, "The path to the private key that is used when authenticating to GitLab", ConfigPropertyAccessMode.READ_WRITE),
```

Comment:
- set a default value
- set property type to just `PropertyType.STRING` (doesn't need to be encrypted)
- tweak description slightly

```suggestion
    GITLAB_JWKS_PATH("integrations", "gitlab.jwks.path", "/oauth/discovery/keys", PropertyType.STRING, "The path to the GitLab instance's JWKS endpoint", ConfigPropertyAccessMode.READ_WRITE),
```